src/components/taskDetails/task-details.module.scss

```diff
@@ -20,6 +20,10 @@ $base-unit: 4px;
     font-weight: bold;
 }
 
+.taskDetailsContainer {
+    margin: 2rem 6rem;
+}
+
```

Comment:
there is an existing class of parentContainer which sets margin for loaded data, can we reuse that instead of creating new class?

Reply:
1. Made this mobile compatible.
2. Made use of the existing class parentContainer

Reply:
thanks for the changes!